.config/tsaoptions.json

```diff
@@ -0,0 +1,10 @@
+{
+    "instanceUrl": "https://devdiv.visualstudio.com/",
+    "template": "TFSDEVDIV",
+    "projectName": "DEVDIV",
+    "areaPath": "DevDiv\\NET Fundamentals\\.NET Acquisition",
```

Comment:
I think we should consider using a subpath for deployment-tools so that we can have unique queries for the various parts of the A&D team.

Reply:
> I think we should consider using a subpath for deployment-tools so that we can have unique queries for the various parts of the A&D team.

That's a good idea - I'll create a sub-path `Deployment-tools` and update the PR.

Reply:
> I think we should consider using a subpath for deployment-tools so that we can have unique queries for the various parts of the A&D team.

Actually we have the following path: `DevDiv\NET Fundamentals\.NET Acquisition\Tools and Infrastructure`

@MichaelSimons are you OK with this or should we create a different one?

Reply:
I think we should create one specific to this repo.  It will be more understandable.